Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -2328,30 +2333,28 @@ def splunk_job_status(service: client.Service, args: dict) -> CommandResults | N
         )
 
 
-def splunk_parse_raw_command():
-    raw = demisto.args().get('raw', '')
+def splunk_parse_raw_command(args: dict):
+    raw = args.get('raw', '')
     rawDict = rawToDict(raw)
-    ec = {}
-    ec['Splunk.Raw.Parsed'] = rawDict
+    ec = {'Splunk.Raw.Parsed': rawDict}
     demisto.results({"Type": 1, "ContentsFormat": "json", "Contents": json.dumps(rawDict), "EntryContext": ec})
 
 
-def test_module(service: client.Service) -> None:
+def test_module(service: client.Service, params: dict) -> None:
     try:
         # validate connection
         service.info()
     except AuthenticationError:
         return_error('Authentication error, please validate your credentials.')
 
     # validate fetch
-    params = demisto.params()
     if params.get('isFetch'):
-        t = datetime.utcnow() - timedelta(hours=1)
+        t = datetime.now(timezone.utc) - timedelta(hours=1)
         time = t.strftime(SPLUNK_TIME_FORMAT)
         kwargs = {'count': 1, 'earliest_time': time}
         query = params['fetchQuery']
         try:
-            if MIRROR_DIRECTION.get(params.get('mirror_direction')) and not params.get('timezone'):
+            if MIRROR_DIRECTION.get(params.get('mirror_direction', '')) and not params.get('timezone'):
```

Comment:
why? both `None` and `''` are falsy.

Reply:
Argument 1 to "get" of "dict" has incompatible type "Optional[Any]"; expected "str" mypy